docs/CrossAccount.md

```diff
@@ -57,6 +57,31 @@ following to buckets in your *DataAccount*.
     ]
 }
 ```
+## Notifications
+
+In order for *ControlAccount* to use an existing and
+[correctly configured SNS topic](EventBridge.md#the-workarounds)
+for a single bucket in *DataAccount*, add a statement similar to the following
```

Comment:
might make sense to emphasize that 'add' means to leave existing statements :blush: